test/headless/specs/intersection-observer.js

```diff
@@ -135,7 +135,7 @@ testModule('IntersectionObserver', class extends TestClass {
     .wait(IMPRESSION_THRESHOLD)
     .scrollTo(105)
     .wait(IMPRESSION_THRESHOLD)
-    .scrollTo(95)
+    .scrollTo(40)
```

Comment:
I believe there is something wrong with way electron calculate the rootMargin. It's just a temporary fix. Need to find a way to use spaniel intersectionObserver  rather native intersection observer.